compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -509,7 +509,7 @@ convType env =
                 tyvar = HsTyVar noExt NotPromoted . noLoc
                     . mkOrig ghcMod . mkOccName clsName $ T.unpack tyname
             args <- mapM (convType env) lfArgs
-            pure $ foldl (HsAppTy noExt . noLoc) tyvar (map noLoc args)
+            pure $ HsParTy noExt (noLoc $ foldl (HsAppTy noExt . noLoc) tyvar (map noLoc args))
```

Comment:
Not entirely sure this is necessary given the context where typeclasses appear but `HsAppTy` definitely needs parentheses in some cases so I’d rather just always add them.

Reply:
Sounds good, & might as well.